lorax.spec

```diff
@@ -99,6 +114,11 @@ make DESTDIR=$RPM_BUILD_ROOT mandir=%{_mandir} install
 %{_datadir}/lorax/*
 %{_mandir}/man1/*.1*
 
+%files composer
+%{python_sitelib}/pylorax/api/*
+%{_sbindir}/lorax-composer
+%{_prefix}/lib/systemd/system/lorax-composer.service
```

Comment:
Related to above, we can use %{_unitdir}/lorax-composer.service, which does appear to be valid for el7